docs/how-to/redeployment.md

```diff
@@ -0,0 +1,37 @@
+# Redeployment Guide
+
+This guide provides the steps necessary for migrating an existing WordPress
+instance to a new charm instance.
+
+## Migrate Database
+
+Follow the instructions
+in [the mysql charm migration guide](https://charmhub.io/mysql/docs/h-migrate-cluster-via-restore)
```

Comment:
I thought `h-migrate-cluster-via-restore` was a typo but it seems it isn't :joy: